validator_client/src/cli.rs

```diff
@@ -27,14 +27,15 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true),
         )
         .arg(
-            Arg::with_name("disable-run-on-all")
```

Comment:
Usually we leave flags around for a version or two so that we don't break people's setups. We could either make `--disable-run-on-all` behave the same as `--broadcast none`, or just log a deprecation warning (which is required in both cases).

Reply:
Good catch!